src/test/suite/views/webview-app/overview-page.test.tsx

```diff
@@ -23,59 +25,57 @@ describe('OverviewPage test suite', function () {
     ).to.exist;
   });
 
-  test('on click of resources, it should open resources panel', function () {
+  it('should open and close the resources panel', async function () {
     render(<OverviewPage />);
-    screen.getByText('Resources').click();
+
+    expect(screen.queryByText('Product overview')).to.not.exist;
+    await userEvent.click(screen.getByText('Resources'));
     expect(screen.getByText('Product overview')).to.exist;
-  });
 
-  test('on click of close button on resources panel, it should close resources panel', function () {
-    render(<OverviewPage />);
-    screen.getByText('Resources').click();
-    screen.getByLabelText('Close').click();
+    await userEvent.click(screen.getByLabelText('Close'));
     expect(screen.queryByText('Product overview')).to.be.null;
   });
 
   describe('Connection Form', function () {
-    test('it is able to open and close the new connection form', function () {
+    it('is able to open and close the new connection form', async function () {
       render(<OverviewPage />);
 
       expect(screen.queryByTestId(connectionFormTestId)).to.not.exist;
-      const postMessageSpy = Sinon.spy(vscode, 'postMessage');
+      const postMessageSpy = sinon.spy(vscode, 'postMessage');
       expect(postMessageSpy).to.not.be.called;
 
-      screen.getByText('Open form').click();
+      await userEvent.click(screen.getByTestId('open-connection-form-button'));
```

Comment:
Opening the connection form takes ~4 seconds for all of the animations to complete. Going to see if this is something we can fix on the connection form side. There's also an issue with focus-trap not finding the fallback node. It's likely fixed in newer versions, which we'll get with newer ConfirmationModals so I'm going to try that as well.

If the delay can't go away then we can skip the animation waiting  (await -> void) and just insert a quick fake delay here with a comment.

Reply:
Used the `tabbable` register jsdom fix/workaround we also use in Compass. Still getting that slow connection form rendering. Haven't dug into it yet. It does slow down tests which is a bit painful. I don't notice it when running the extension though.